vignettes/dust.Rmd

```diff
@@ -297,6 +297,55 @@ matplot(time, state, type = "l", lty = 1, col = "#00000022",
         xlab = "Time", ylab = "Number infected (I)")
 ```
 
+You can optionally set the initial step at the same time as the state. This is useful in two cases; if your model depends on time (e.g., you use the step in a calculation by transforming it into some measure of time) or if your particles start at different points. For example, suppose we start 10 individuals in the infected state but we do so with a number of possible starting points sampled from between step 0 and step 30:
+
+```{r initial_step}
+step0 <- sample(0:30, 20, replace = TRUE)
+model <- sir$new(list(), 0, 20)
+model$set_state(c(1000, 10, 0), step0)
+```
+
+After initialisation, our model is now at `r max(step0)`, the last step referenced in `step0`
+
+```{r}
+model$step()
+```
+
+and the initial states are a mix of states, despite having been seeded with the same 10 infected individuals, as different particles have run for the same time to reach this time point:
```

Comment:
different particles have run for different times to reach this time point?

Reply:
🤦

Reply:
Fixed in ae39ed6